### DISCUSSION TITLE
There is an officially supported Thunderbird flatpak

### DISCUSSION THREAD

**Discussion by hellsworth** [linuxmint/discussions#377] [Ideas]
Hi All, I hear the Mint team is planning to (perhaps already) provide your own Thunderbird deb package. I know deb packaging can be a pain and Thunderbird is a beast. [It was suggested on mastodon](https://fosstodon.org/@ThePlant@mastodon.social/112214293220896993) that perhaps you weren't aware that we (the Thunderbird team) have an [officially supported flatpak on flathub](https://flathub.org/apps/org.mozilla.Thunderbird). If that's the case, this is just a plug to use our flatpak rather than maintaining your own deb, so as to lighten your load a bit :)

**Comment by mtwebster** [comment]
Hi, thanks for reaching out - we've already begun building debs for our next release.

We considered recommending the Flatpak as the official answer to Ubuntu switching to a snap, but in the end decided to package it ourselves - a decent portion of our user base isn't interested in using Flatpaks (or anything else non-system-packaged) for one reason or another.

For a couple of years now, our Firefox builds have (thankfully!) been simplified to downloading the release tarball and its translations and repackaging it so everything goes to the correct location. This seems to be working for Thunderbird also, so far.

**Reply by hellsworth** [reply]
Totally understandable. I'm happy you have a solution that works for your users 👍
